### PR TITLE
Feature/empty state classes

### PR DESCRIPTION
**Pull request by emclaug2**
Changes proposed in this Pull Request:
- Add class overrides to the EmptyState component
- classes: [root, icon, title, description, actions]

### CODE REVIEW

File: components/src/core/EmptyState/EmptyState.tsx [hunk 1]
```diff
@@ -24,36 +32,49 @@ const useStyles = makeStyles({
         textAlign: 'center',
         alignItems: 'center',
     },
+    icon: {
+        marginBottom: 15,
```
Comment (joebochill) [inline components/src/core/EmptyState/EmptyState.tsx]: `theme.spacing(2)` ?

File: components/src/core/EmptyState/EmptyState.tsx [hunk 2]
```diff
@@ -24,36 +32,49 @@ const useStyles = makeStyles({
...
+        display: 'flex',
+        fontSize: 100,
+    },
+    actions: {
+        marginTop: 10,
```
Comment (joebochill) [inline components/src/core/EmptyState/EmptyState.tsx]: `theme.spacing(1)` ?

File: docs/EmptyState.md [hunk 3]
```diff
@@ -21,9 +21,21 @@ import { EmptyState } from '@pxblue/react-components';
 
 | Prop Name   | Description                             | Type              | Required | Default |
 |-------------|-----------------------------------------|-------------------|----------|---------|
-| title       | The main text to display                | `string`          | yes      |         |
-| description | The secondary text to display           | `string`          | no       |         | 
-| icon        | The primary icon                        | `React.Component` | yes      |         |
 | actions     | Additional components to render below   | `React.Component` | no       |         |
+| classes     | Style overrides                         | `StyleRules`      | no       |         |
```
Comment (joebochill) [inline docs/EmptyState.md]: can you alphabetize these please?

File: components/src/core/EmptyState/EmptyState.tsx (hunk 1 above)
Comment (emclaug2) [reply components/src/core/EmptyState/EmptyState.tsx]: Updated

File: docs/EmptyState.md (hunk 3 above)
Comment (emclaug2) [reply docs/EmptyState.md]: I kept the root at the top of the list, thoughts?

File: components/src/core/EmptyState/EmptyState.tsx (hunk 2 above)
Comment (emclaug2) [reply components/src/core/EmptyState/EmptyState.tsx]: Updated

File: docs/EmptyState.md (hunk 3 above)
Comment (joebochill) [reply docs/EmptyState.md]: yeah that's probably fine

Comment (joebochill) [reply docs/EmptyState.md]: That's cool, but can we alphabetize the remainder?

Comment (emclaug2) [reply docs/EmptyState.md]: Yep, it's updated now.  I'm going to resolve this.

File: docs/EmptyState.md [hunk 4]
```diff
@@ -21,9 +21,21 @@ import { EmptyState } from '@pxblue/react-components';
 
 | Prop Name   | Description                             | Type              | Required | Default |
 |-------------|-----------------------------------------|-------------------|----------|---------|
-| title       | The main text to display                | `string`          | yes      |         |
-| description | The secondary text to display           | `string`          | no       |         | 
-| icon        | The primary icon                        | `React.Component` | yes      |         |
 | actions     | Additional components to render below   | `React.Component` | no       |         |
+| classes     | Style overrides                         | `StyleRules`      | no       |         |    
+| icon        | The primary icon                        | `React.Component` | yes      |         |
+| description | The secondary text to display           | `string`          | no       |         |
```
Comment (joebochill) [inline docs/EmptyState.md]: doesn't d come before i? :)

Comment (emclaug2) [reply docs/EmptyState.md]: Fixed now :)

**Pull request merged**